backend-api/src/functions/asyncBiometricToken/getParsedRequestBody/getParsedRequestBody.ts

```diff
@@ -0,0 +1,100 @@
+import { errorResult, Result, successResult } from "../../utils/result";
+
+export function getParsedRequestBody(
+  body: string | null,
+): Result<IAsyncBiometricTokenValidParsedRequestBody> {
+  if (body == null) {
+    return errorResult({
+      errorMessage: `Request body is either null or undefined. Request body: ${body}`,
+      errorCategory: "CLIENT_ERROR",
```

Comment:
No need for err cat as it's validation of what client has sent, always going to be client error